pygrams.py

```diff
@@ -57,7 +57,9 @@ def get_args(command_line_arguments):
     parser.add_argument("-st", "--search_terms", type=str, nargs='+', default=[],
                         help="Search terms filter: search terms to restrict the tfidf dictionary. "
                              "Outputs will be related to search terms")
-
+    parser.add_argument("-thres", "--search_terms_threshold", type=float, nargs='+', default=0.75,
```

Comment:
To make the abbreviated command line are more obvious?
```suggestion
    parser.add_argument("-stthresh", "--search_terms_threshold", type=float, nargs='+', default=0.75,
```